backend/src/main/java/com/sidedish/domain/EventBadge.java

```diff
@@ -0,0 +1,15 @@
+package com.sidedish.domain;
+
+public enum EventBadge {
+	NONE(1), LAUNCHING(0.8), EVENT(0.9);
+
+	private final double discountAmount;
+
+	EventBadge(double discountAmount) {
```

Comment:
`final` 이면 더 좋겠네요. 왜 일까요?
```suggestion
	EventBadge(final double discountAmount) {
```

Reply:
해당 파라미터가 local 영역에서 변경 될 수 있는 위험이 존재하기 때문에 이렇게 설정해주는 것으로 이해했습니답